src/sentry/models/exporteddata.py

```diff
@@ -65,7 +73,25 @@ def finalize_upload(self, file, expiration=DEFAULT_EXPIRATION):
         current_time = timezone.now()
         expire_time = current_time + expiration
         self.update(file=file, date_finished=current_time, date_expired=expire_time)
-        # TODO(Leander): Implement email notification
+        self.email_user()
+
+    def email_user(self):
+        from sentry.utils.email import MessageBuilder
+
+        if self.date_finished is None or self.date_expired is None or self.file is None:
+            return
```

Comment:
I'm not sure if this should throw an error (since someone would have to be prompting an email with an imcomplete export). It _shouldn't_ ever even trigger, but its a safeguard

Reply:
I'd put the fact that this is a safeguard in a comment. I think a `return` here should be fine. Maybe also not a bad idea to add a log line if this does happen.

Reply:
I'll be doing logging in a separate PR, but the comment is in 28c0027